.deployment/barista/internal/nginx/sites-available/barista-internal.conf

```diff
@@ -18,13 +18,6 @@ server {
     deny all;
   }
 
-  location ~ ^/next(/)? {
-    root /var/www/next/;
-    # https://stackoverflow.com/a/41629576
-    rewrite  ^/next/(.*) /$1 break;
-    try_files $uri /next/index.html;
-  }
-
```

Comment:
we might need this in the future again i guess - but we have it in the history

Reply:
no as this with the rewrite is only for SPAs and we will have static sites it will be different :)